gef.py

```diff
@@ -6157,6 +6166,28 @@ def do_invoke(self, _: List[str], **kwargs: Any) -> None:
         return
 
 
+@register
+class GnextiCommand(GenericCommand):
+    """Step over call instruction"""
```

Comment:
I also find this very useful for stepping over instructions with `rep` prefix.

Reply:
done!